.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": [
+    "react",
+    "plugin:prettier/recommended"
+  ]
+}
```

Comment:
Try to add a trailing line to this files to avoid the `No newline error`

Reply:
This entry `insert_final_newline = true` in .editorconfig will solve that on all files

Reply:
Cool, that is subsequently. But for this PR, we should solve all of that